source/a_releasenotes.rst

```diff
@@ -0,0 +1,65 @@
+.. raw:: html
+
+    <style> .blue {color:#4a86e8} </style>
+
+.. role:: blue
```

Comment:
Die role:blue stond in de vorige file ook. Hier staat er ook een style bij. Dat gaat goed in de vorige file?